mappings/1.8/net/minecraft/entity/living/attribute/AbstractEntityAttributeContainer.mapping

```diff
@@ -1,8 +1,11 @@
 CLASS net/minecraft/C_85mefkhti net/minecraft/entity/living/attribute/AbstractEntityAttributeContainer
 	FIELD f_48ybdxuqi byName Ljava/util/Map;
 	FIELD f_79vrbdekb byType Ljava/util/Map;
+	FIELD f_90yrtmncs attributesMap Lcom/google/common/collect/Multimap;
```

Comment:
imo the `Map` suffix is either redundant, or should be replaced by `By<key>` where `key` is the type of the key of the map.